File: VB/WindowsApplication3/Custom Ribbon/MyRibbonBarManager.vb
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.ViewInfo

Namespace DXSample

    Public Class MyRibbonBarManager
        Inherits RibbonBarManager

        Public Sub New(ByVal ribbonControl As RibbonControl)
            MyBase.New(ribbonControl)
        End Sub

        Friend Overloads ReadOnly Property SelectionInfo As BarSelectionInfo
            Get
                Return MyBase.SelectionInfo
            End Get
        End Property
    End Class
End Namespace

File: CS/WindowsApplication3/Custom Ribbon/MyRibbonBarManager.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;
using DevExpress.Skins;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.ViewInfo;

namespace DXSample {
    public class MyRibbonBarManager : RibbonBarManager
    {
        public MyRibbonBarManager(RibbonControl ribbonControl) : base(ribbonControl) { }
        internal new BarSelectionInfo SelectionInfo
        {
            get { return base.SelectionInfo; }
        }
    }
}